apps/docs/content/references/import.mdx

```diff
@@ -98,23 +108,29 @@ services:
           access_log syslog:server=unix:/dev/log,facility=local1 default_short;
           error_log syslog:server=unix:/dev/log,facility=local1;
       }
-    # zerops.yaml service name or full config file
-    zeropsSetup:
-      build:
-        base: nodejs@latest
-        buildCommands:
-          - echo zerops.yaml from import
-          - yarn
-          - yarn run build
-        deployFiles: ./
-        cache: node_modules
-      run:
-        initCommands:
-          - |
-            if ! zcli bucket s3 create $STORAGE_HOSTNAME $STORAGE_BUCKET_NAME --x-amz-acl=private; then
-            echo "The bucket was not created, you have to do it manually!"
-            fi
-        start: yarn start
+    # Service name from zerops.yaml or reference to a service setup
+    zeropsSetup: nodejs-app
+    # Full zerops.yaml configuration
+    zeropsYaml:
+      zerops:
+        - setup: nodejs-app
```

Comment:
tomuhle bych se vyhnul, nepodporujeme `-` v hostname servis, teoreticky to jako setup můžeš použít v kombinaci právě se zeropsSetup, nebo když používáš `extends`, ale asi lepší nedávat lidem nápady